cmds/provisiond/main.go

```diff
@@ -51,6 +52,14 @@ func main() {
 		log.Logger.Level(zerolog.DebugLevel)
 	}
 
+	// keep checking if limited-cache flag is set
+	if app.CheckFlag("limited-cache") {
+		log.Info().Msg("failed cache reservation! Retrying every 30 seconds...")
```

Comment:
Let's log using Error() instead of info so it's easier for us to grep on the logs.

Reply:
Please also use the defined constant, not string "limited-cache"

Reply:
fixed